README.md

```diff
@@ -122,9 +122,11 @@ We have other projects tightly related to ArkScript, which aren't necessarily C+
 
 [Full list here](https://github.com/ArkScript-lang/Ark/graphs/contributors).
 
-### Contributing to the ArkScript standard library
+### Coding guidelines for contributing
 
-See [Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
+See [C++ Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
```

Comment:
The link is for C++, thus "if you want to write ArkScript for the library (see folder `lib/std/`)." should be more like "if you want to contribute to ArkScript compiler / runtime".